setup.py

```diff
@@ -258,7 +258,7 @@ def get_exts_for(name):
         "protobuf>=3; python_version>='3.7'",
         "protobuf>=3,<4.0; python_version=='3.6'",
         "protobuf>=3,<3.18; python_version<'3.6'",
-        "tenacity>=5",
+        "tenacity>=7",
```

Comment:
Do we really need to make this change? We don't actually require it for our usage.

Users using Python 3.11 will have a higher version of tenacity anyway, we just need to set a different minimum version used for our profiler 3.11 tests.

Reply:
We don't stricto sensu, I'll revert it.